packages/common/src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -1513,11 +1513,10 @@ export class AudiusAPIClient {
     }, {})
 
     const formattedPath = this._formatPath(pathType, path)
-    const audiusLibs =
-      this.audiusLibs ??
-      (this.initializationState.type === 'libs' && window.audiusLibs)
-    if (audiusLibs) {
-      const data = await audiusLibs.discoveryProvider._makeRequest(
+    const audiusLibs = this.getAudiusLibs()
```

Comment:
So in this case we aren't awaiting?

Reply:
I think the diff might look a little funky, but no we aren't awaiting libs because we want to make the request eagerly before it is initialized. `_makeRequest` is still awaited if audiusLibs exists